src/components/SliderMenu.tsx

```diff
@@ -1,48 +1,98 @@
-import { Button, type DisplayTheme } from "@arconnect/components";
+import { type DisplayTheme } from "@arconnect/components";
 import { CloseIcon } from "@iconicicons/react";
+import { AnimatePresence, motion, type Variants } from "framer-motion";
+import type React from "react";
+import { useRef } from "react";
 import styled, { useTheme } from "styled-components";
-import { SendInput } from "~routes/popup/send";
 
 interface SliderMenuProps {
   title: string;
+  isOpen: boolean;
   onClose?: () => void;
   children?: React.ReactNode;
 }
 
 export default function SliderMenu({
-  children,
   title,
-  onClose
+  isOpen,
+  onClose,
+  children
 }: SliderMenuProps) {
+  const wrapperElementRef = useRef<HTMLDivElement | null>(null);
   const theme = useTheme();
-  return (
-    <Wrapper displayTheme={theme.displayTheme}>
-      <Body>
-        <Header>
-          <Title>{title}</Title>
-          <ExitButton onClick={onClose} />
-        </Header>
-        {children}
-      </Body>
-    </Wrapper>
-  );
+
+  const contentElement = isOpen ? (
+    <>
+      <CloseLayer
+        key="SliderMenuCloseLayer"
+        exit={{ opacity: 0 }}
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        transition={{ duration: 0.2 }}
+        onClick={(e) => {
+          e.stopPropagation();
+          onClose();
+        }}
+      />
+
+      <Wrapper
+        displayTheme={theme.displayTheme}
+        ref={wrapperElementRef}
+        variants={animationSlideFromBottom}
+        initial="hidden"
+        animate="shown"
+        exit="hidden"
+      >
+        <Body>
+          <Header>
+            <Title>{title}</Title>
+            <ExitButton onClick={onClose} />
+          </Header>
+          {children}
+        </Body>
+      </Wrapper>
+    </>
+  ) : null;
+
+  return <AnimatePresence>{contentElement}</AnimatePresence>;
 }
 
 const ExitButton = styled(CloseIcon)`
   cursor: pointer;
 `;
 
-const Wrapper = styled.div<{ displayTheme: DisplayTheme }>`
+const Wrapper = styled(motion.div)<{ displayTheme: DisplayTheme }>`
   position: fixed;
+  bottom: 0;
+  left: 0;
+  height: 93%;
```

Comment:
@7i7o hmm - I think it was more so my preference, I'll move it back to auto but I changed it because a low sliding slider didn't feel balanced to me 

<img width="372" alt="image" src="https://github.com/user-attachments/assets/5e7f7fa2-8718-4909-95d7-c18d5b62c2da">

Reply:
Yeah, I agree that it doesn't look so good in this case, but I guess that's because it also ends in a heading. Maybe we could create a task to put a `<No Contacts>` text/link to add some or something similar so it's not that unbalanced